docs/change_log/2_0_0_rc1.md

```diff
@@ -3,9 +3,11 @@
 ```eval_rst
 .. admonition:: v1.x升级到v2.0.0-rc1
 
-    - **v2.0.0-rc2不兼容v1.x，v2.0.0-rc1无法直接解析v1.x产生的历史区块数据** ，但可通过在v2.0.0-rc2的新链上执行历史交易的方式恢复旧数据
+    - **v2.0.0-rc2不兼容v1.x，v2.0.0-rc1无法直接解析v1.x产生的历史区块数据** ，但可通过在 `v2.0.0-rc1 <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/download/v2.0.0-rc1/fisco-bcos.tar.gz>`_ 的新链上执行历史交易的方式恢复旧数据
 
     - **搭建2.0的新链** ：请参考 `安装 <../installation.html>`_ 
+
+    - `v2.0.0-rc1 release <https://github.com/FISCO-BCOS/FISCO-BCOS/releases/tag/v2.0.0-rc1>`_
```

Comment:
v2.0.0-rc1 release notes

Reply:
already modified from `release note` to `Relase Note`.